lagrangebench/case_setup/case.py

```diff
@@ -63,13 +64,12 @@ def case_builder(
     box: Tuple[float, float, float],
     metadata: Dict,
     input_seq_length: int,
-    isotropic_norm: bool = defaults.isotropic_norm,
-    noise_std: float = defaults.noise_std,
+    cfg_neighbors: DictConfig = defaults.neighbors,
+    isotropic_norm: bool = defaults.model.isotropic_norm,
+    noise_std: float = defaults.train.noise_std,
     external_force_fn: Optional[Callable] = None,
-    magnitude_features: bool = defaults.magnitude_features,
-    neighbor_list_backend: str = defaults.neighbor_list_backend,
-    neighbor_list_multiplier: float = defaults.neighbor_list_multiplier,
-    dtype: jnp.dtype = defaults.dtype,
+    magnitude_features: bool = defaults.model.magnitude_features,
+    dtype: jnp.dtype = defaults.main.dtype,
```

Comment:
why did you leave these as regular args?

Reply:
No particular reason. I don't want to pass the full configs if only 1-2 things are used.